RELEASE/scripts/autoscend.ash

```diff
@@ -2351,37 +2351,44 @@ int speculative_pool_skill()
 
 boolean autosellCrap()
 {
-	if((item_amount($item[dense meat stack]) > 1) && (item_amount($item[dense meat stack]) <= 10))
+	if(can_interact() && my_meat() > 20000)
 	{
-		auto_autosell(1, $item[dense meat stack]);
+		return false;		//do not autosell stuff in casual or postronin unless you are very poor
 	}
-	foreach it in $items[Blue Money Bag, Red Money Bag, White Money Bag]
+	
+	foreach it in $items[dense meat stack, meat stack, Blue Money Bag, Red Money Bag, White Money Bag]
 	{
 		if(item_amount(it) > 0)
 		{
-			auto_autosell(item_amount(it), it);
+			auto_autosell(item_amount(it), it);		//autosell all of this item
```

Comment:
I think we should autosell whatever is smaller of 20 and item_amount(it) to protect from weird edgecases like maxint meat or people who keep their entire meat savings in dense meat stacks for other reasons

Reply:
added limit of 10 per sell